e2etest/newe2e_task_validation.go

```diff
@@ -191,3 +194,110 @@ func checkMultipleErrors(errorMsg []string, line string) bool {
 
 	return false
 }
+
+func ValidateListTextOutput(a Asserter, stdout AzCopyStdout, expectedObjects map[AzCopyOutputKey]cmd.AzCopyListObject, expectedSummary *cmd.AzCopyListSummary) {
```

Comment:
This works, but could be better.

Consider this-- Parse the output line by line, compare to the map that already exists. If any elements of the map have not been hit, or any of the elements of the output miss the map, log a failure.